### DISCUSSION TITLE
BIDS App wrapper for HCP Pipelines v4.3.0

### DISCUSSION THREAD

**Discussion by giulia-berto** [Washington-University/HCPpipelines#208] [Q&A] [answered]
Will you release a BIDS App wrapper for the new version of HCP Pipelines v4.3.0? The last one I can find is related to v4.1.3.

**Comment by mharms** [comment, marked as answer]
We are not the creator/maintainer of the BIDS App for the HCPpipelines.

**Comment by glasserm** [comment]
Please contact whomever is currently doing that.  Soon there will be an official container for the HCP Pipelines called Qu|Nex.

**Reply by giulia-berto** [reply]
ok thanks!
